### ISSUE TITLE
Showing ScrollBar < and > buttons

### ISSUE THREAD

**Issue by martinm1000** [JFormDesigner/FlatLaf#25]
Hi, is there a way or option to have the scrollbar prev/next button showing ?

I have a heavily customized JTable where in that case the prev < and > next button of the scrollbar were used to move by a specific amount.

**Comment by DevCharly** [comment]
This is now implemented in master branch.

To show prev/next buttons for all scrollbars in your application use:
~~~java
UIManager.put( "ScrollBar.showButtons", true );
~~~

Or enable them on single components with:
~~~java
myScrollPane.putClientProperty("JScrollBar.showButtons", true);
~~~
or
~~~java
myScrollBar.putClientProperty("JScrollBar.showButtons", true);
~~~

**Comment by martinm1000** [comment]
Thank you very much !

**Comment by JFormDesigner** [comment]
released in 0.21
